components/abstractions/src/main/java/com/microsoft/kiota/serialization/KiotaSerialization.java

```diff
@@ -29,7 +30,26 @@ private KiotaSerialization() {}
      */
     @Nonnull public static <T extends Parsable> InputStream serializeAsStream(
             @Nonnull final String contentType, @Nonnull final T value) throws IOException {
-        try (final SerializationWriter writer = getSerializationWriter(contentType, value)) {
+        return serializeAsStream(contentType, DEFAULT_SERIALIZE_ONLY_CHANGED_VALUES, value);
+    }
+
+    /**
+     * Serializes the given value to a stream and configures returned values by the backing store if available
+     * @param <T> the type of the value to serialize
+     * @param contentType the content type to use for serialization
+     * @param value the value to serialize
+     * @param serializeOnlyChangedValues whether to serialize all values in value if value is a BackedModel
+     * @return the serialized value as a stream
+     * @throws IOException when the stream cannot be closed or read.
+     */
+    @Nonnull public static <T extends Parsable> InputStream serializeAsStream(
+            @Nonnull final String contentType,
+            final boolean serializeOnlyChangedValues,
```

Comment:
should we make the boolean parameter last so that if a refator is done from the previous method, it will seem like a new parameter is added at the end of the argument list rather than at the middle?

Reply:
Good point, followed the linting warning a little blindly. Adding a linting option to disable it since it doesn't apply in this case.